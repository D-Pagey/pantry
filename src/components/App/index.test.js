import React from 'react';
import { App } from '.';

describe('App component', () => {
  it('should render', () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
