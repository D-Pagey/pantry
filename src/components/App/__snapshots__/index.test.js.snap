// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 0 0 2rem;
  padding: 2rem 0 0;
}

.c2 {
  color: initial;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3 {
  font-weight: 500;
  margin: 0;
}

.c4 {
  background-color: grey;
  bottom: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  list-style: none;
  padding: 0;
  left: 0;
  margin: 0;
  position: fixed;
  right: 0;
}

.c5 {
  width: 50%;
}

.c6 {
  color: black;
  display: inline-block;
  padding: 1rem 0;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  width: 100%;
}

.c6.active {
  background-color: white;
}

.c0 {
  margin: 0 auto;
  max-width: 1250px;
  padding: 0 1rem 2rem;
}

<div
  class="c0"
>
  <header
    class="c1"
  >
    <a
      aria-current="page"
      class="c2 active"
      href="/"
    >
      <h4
        class="c3"
      >
        Pantry
      </h4>
    </a>
  </header>
  <div
    class=""
  >
    <button>
      Sign In
    </button>
    <p>
      Loading...
    </p>
  </div>
  <ul
    class="c4"
    data-testid="mobileNavbar"
  >
    <li
      class="c5"
    >
      <a
        aria-current="page"
        class="c6 active"
        href="/"
      >
        Check Fridge
      </a>
    </li>
    <li
      class="c5"
    >
      <a
        class="c6"
        href="/add"
      >
        Add Food
      </a>
    </li>
  </ul>
</div>
`;
